.github/custard/map.sh

```diff
@@ -0,0 +1,27 @@
+set -e # Exit on error
+set -u # Error when expanding unset variables
+
+SCRIPT="$1"
+JSON_LIST="$2"
+
+SUCCEEDED=""
+FAILED=""
+for path in $(echo "$JSON_LIST" | jq -r '.[]'); do
```

Comment:
jq comes pre-installed in the GitHub actions machines, but it might not be available locally.

Another option here would be to write this in JS, but we would now have to depend on node. It might be safe to assume having all three languages as part of the development cycle on this repo anyways.

Reply:
FWIW, it seems OK to require `jq` locally as long as it's documented.

Reply:
Last minute change, I decided to make this script take in positional arguments. It's easier to use that way too. The GHA workflow will still use jq to transform the JSON list into positional arguments. That way we don't get an extra implicit dependency